app/src/docs/docsEditingGuide.md

```diff
@@ -0,0 +1,52 @@
+# Docs Editing Guide
```

Comment:
Should we link to this file from the app readme?

Reply:
yes, good idea, I will add